packages/@aws-cdk/aws-fsx/lib/lustre-file-system.ts

```diff
@@ -50,6 +50,21 @@ export enum LustreAutoImportPolicy {
   NEW_CHANGED_DELETED = 'NEW_CHANGED_DELETED',
 }
 
+/**
+  * The permitted Lustre data compression algorithms
+*/
+export enum LustreDataCompressionType {
+  /**
+  *
+  * `NONE` - (Default) Data compression is turned off when the file system is created.
+  */
+  NONE = 'NONE',
+  /**
+  * `LZ4` - Data compression is turned on with the LZ4 algorithm.
```

Comment:
Can you add a note in here that it only acts on new files, not existing? Not everyone checks the README so I want to make sure it's clear in the docstring.

Reply:
Done